src/coreclr/vm/riscv64/stubs.cpp

```diff
@@ -1507,14 +1515,78 @@ void StubLinkerCPU::EmitCallManagedMethod(MethodDesc *pMD, BOOL fTailCall)
 //
 // Allocation of dynamic helpers
 //
+enum Riscv64_registers
+{
+    REG_ZERO,
```

Comment:
We usually use `REG_R0`.

Reply:
Done, now this form of register writing is used `const IntReg RegR0`